lib/ocpp/v201/charge_point.cpp

```diff
@@ -209,8 +209,8 @@ void ChargePoint::start(BootReasonEnum bootreason) {
     // get transaction messages from db (if there are any) so they can be sent again.
     this->message_queue->get_persisted_messages_from_db();
     this->boot_notification_req(bootreason);
-    // K01.27 - call load_charging_profiles when system boots
-    this->load_charging_profiles();
+    // call clear_invalid_charging_profiles when system boots
```

Comment:
This comment is only describing what is happening. It's more useful if you add the why, otherwise I think its better to remove the comment and let the code do the talking ;)